woocommerce/woocommerce
# Add coming soon template - split right image

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #52979.

This PR adds a new Coming Soon template- split right image, built using WP blocks. The template follows the Figma design and ensures that all elements, including images and copy, are fully editable to provide flexibility for customization.

I had to add some custom CSS to make the template match the Figma design because Blocks Editor doesn't support it. For example, the cover block set text to center, but we want it to be top-aligned.

![Screenshot 2024-12-17 at 11 45 44](https://github.com/user-attachments/assets/edad9bc4-bdd6-4949-b123-14fcf1206151)

![Screenshot 2024-12-17 at 11 46 49](https://github.com/user-attachments/assets/03502ab8-109d-47e0-b2dd-fe5a79c2e283)

![Screenshot 2024-12-17 at 11 54 21](https://github.com/user-attachments/assets/d875ec14-3b5d-4a50-9247-0e69ba0ef95a)




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Create a fresh site with  `coming-soon-newsletter-template` feature flag enabled.
2. Enable coming soon mode in site visibility settings.
3. Go to `Appearance > Editor > Templates > Coming soon`
4. Choose `Coming Soon Split Right Image` template on the right sidebar (Template > Design > Coming Soon Split Right Image)
6. Confirm the template matches the design in Figma.
7. Confirm that you can customize the styles, replace image, change the content, etc.
8. Select `Coming Soon Split Right Image` template again and save the template.
9. Go to the site front e incognito tab to make sure the template is loaded correctly. The page should be responsive.
10. Go to `Appearance > Editor > Templates` and confirm the coming soon template preview shows the selected template.

Figma:  0BWMerqRR1f468EwC7g5tg-fi-3708_131070

![image](https://github.com/user-attachments/assets/c1cda7f7-5b24-4062-a3af-6b723365316f)





### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Add coming soon template - split right image

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
